Sparkle/SUTouchBarForwardDeclarations.h

```diff
@@ -0,0 +1,91 @@
+//
+//  SUTouchBarForwardDeclarations.h
+//  Sparkle
+//
+//  Created by Yuxin Wang on 18/01/2017.
+//  Copyright © 2017 Sparkle Project. All rights reserved.
+//
+
+// Once Sparkle no longer supports OSX 10.12.0, this file can be deleted.
+
+#import <Foundation/Foundation.h>
+
+#if !defined(MAC_OS_X_VERSION_10_12_1)
```

Comment:
It is defined in SDKROOT/usr/include/AvailabitityMacros.h

I took an example of:
https://chromium.googlesource.com/chromium/src/+/master/ui/base/cocoa/touch_bar_forward_declarations.h
https://chromium.googlesource.com/chromium/src/base/+/master/mac/sdk_forward_declarations.h